SPECS/Cython/Cython.spec

```diff
@@ -1,53 +1,85 @@
 %{!?python2_sitelib: %define python2_sitelib %(python2 -c "from distutils.sysconfig import get_python_lib;print(get_python_lib())")}
+%{!?python3_sitelib: %define python3_sitelib %(python3 -c "from distutils.sysconfig import get_python_lib;print(get_python_lib())")}
 Summary:       C extensions for Python
 Name:          Cython
 Version:       0.28.5
-Release:       7%{?dist}
+Release:       8%{?dist}
 Group:         Development/Libraries
 License:       ASL 2.0
 URL:           https://cython.org
 #Source0:      https://github.com/cython/cython/archive/%{version}.tar.gz
 Source0:       %{name}-%{version}.tar.gz
 Patch0:        fix_abc_tests.patch
+Patch1:        cython_testfix_with_outer_raising.patch
 Vendor:        Microsoft Corporation
 Distribution:  Mariner
 BuildRequires: python2-devel
 BuildRequires: python2-libs
 BuildRequires: python-xml
 Requires:      python2
+Provides:      python2-%{name}
```

Comment:
I understand the lower case `cython`, but is `python2-Cython` used anywhere?

Reply:
Talked it out with Jon- Not currently used by us, but there are other distributions which use this name (OpenSUSE, CentOS). We figured it wouldn't hurt to be consistent with our package naming standards here.